wsagent/che-core-git-impl-jgit/src/main/java/org/eclipse/che/git/impl/jgit/JGitConfigImpl.java

```
@@ -108,6 +114,11 @@ public Config add(String name, String value) throws GitException {
     public Config unset(String name) throws GitException {
         ConfigKey key = parseName(name);
         repository.getConfig().unset(key.section, key.subsection, key.name);
+        try {
+            this.repository.getConfig().save();
+        } catch (IOException e) {
+            throw new GitException("Failed to save to config file");
```

as exception is dropped, should we at least log it ?